### PR TITLE
hxrshilgupta's card

### PR DESCRIPTION
**Pull request by hxrshilgupta**
made my first PR contribution to get started
Added my card to  #index.html

### CODE REVIEW

**Review by ibrahimdevop21 — CHANGES_REQUESTED** [review]
Hi there!
Thank you so much for your contribution! We noticed that the URLs in the href attributes are placeholders. Could you please replace the # with the actual URLs before we can merge your pull request?
Once you've made these updates, kindly resubmit your pull request. If you have any questions or need any assistance, feel free to reach out.
Thanks again for your help and looking forward to your updated submission!
Best regaards,
ibrahimdevop21

**Comment by hxrshilgupta** [comment]
Hi @ibrahimdevop21 
Sure I will make the changes right away. Thanks for pointing them out : )

**Comment by ibrahimdevop21** [comment]
Looking forward to 👀 your output

File: index.html [hunk 1]
```diff
@@ -120,7 +120,33 @@ <h2>A project for first-time contributions</h2>
...
+            <a href="https://www.twitter.com/hxrshilgupta" target="_blank">hmu on twitter :)</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/hxrshilgupta" target="_blank">i do github sometimes</a>
+          </p>
+          <p class="about">cse senior @kiituniversity, click <a href="https://bento.me/hxrshilgupta">here</a> to know more about me</p>
+          <div class="resources">
+            <p>i do stuff like</p>
+            <ul>
+              <li>
+                <a href="" target="_blank" title="First Resource">learning new tech things</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:137]: Please add an actual href:

**Pull request merged**